.github/workflows/deploy-on-label.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         echo "Current PR state: $STATE"
     - name: Circle CI Deployment Trigger
       id: curl-circle-ci
-      if: steps.get-pr-state.outputs.STATE == 'success'
+#      if: steps.get-pr-state.outputs.STATE == 'success'
```

Comment:
Was this just for testing in the raft environment to prevent disruptions? Should this be commented back in?

Reply:
Correct, good catch